covid19-dashboard/src/Utils.ts

```diff
@@ -26,14 +26,20 @@ export const numberWithCommas = (num: number): string => {
 };
 
 /**
- * Gets the latest date in the for a a list of Iso-dates.
+ * Gets the min or max date in a list of ISO dates.
```

Comment:
Revert back to getLatestDate

Reply:
Can we do this in a separate PR so that can rename the function in the necessary files?

Reply:
perhaps this PR isn't complete then. I don't see the renamed function used in these files (or it's not obvious anyway). Could you just revert this change?